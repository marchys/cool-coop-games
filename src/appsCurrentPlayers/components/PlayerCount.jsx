import React from 'react';
import PropTypes from 'prop-types';

import styled, { css } from 'react-emotion';

const Container = styled('div')`
  display: flex;
  justify-content: space-around;
  align-items: center;
  background-color: green;
`;

export default function PlayerCount({ appId, playerCount }) {
  return (
    <Container>
      <p
        className={css`
          color: mediumspringgreen;
        `}
      >
        ID: {appId}
      </p>
      <p className={css({ color: 'cornsilk' })}>Count: {playerCount}</p>
    </Container>
  );
}

PlayerCount.propTypes = {
  appId: PropTypes.string.isRequired,
  playerCount: PropTypes.number.isRequired,
};
