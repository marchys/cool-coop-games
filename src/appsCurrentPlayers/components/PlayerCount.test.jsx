import React from 'react';
import { shallow } from 'enzyme';

import PlayerCount from './PlayerCount';

function setup({ appId = '74933', playerCount = 200, ...props } = {}) {
  const wrapper = shallow(<PlayerCount appId={appId} playerCount={playerCount} {...props} />);

  return {
    wrapper,
  };
}

test('should render ', () => {
  const { wrapper } = setup();

  expect(wrapper).toMatchSnapshot();
});
