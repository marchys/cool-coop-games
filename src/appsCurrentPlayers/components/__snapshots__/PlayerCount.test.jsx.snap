// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render  1`] = `
<Styled(div)>
  <p
    className="css-1q0hz49"
  >
    ID: 
    74933
  </p>
  <p
    className="css-1md3z78"
  >
    Count: 
    200
  </p>
</Styled(div)>
`;
